nengo_bones/config.py

```diff
@@ -56,22 +127,27 @@ def validate_config(config):
     config : dict
         Dictionary containing configuration values.
     """
-    mandatory = ["pkg_name", "repo_name"]
-    if "travis_yml" in config:
-        mandatory.append("travis_yml.jobs")
+    mandatory = ["project_name", "pkg_name", "repo_name", "travis_yml.jobs"]
 
     for entry in mandatory:
         tmp = config
-        for key in entry.split("."):
+        for i, key in enumerate(entry.split(".")):
             try:
                 tmp = tmp[key]
             except KeyError:
-                raise KeyError("Config file must define %s" % entry)
+                if "." in entry and i == 0:
+                    # if the toplevel isn't defined, ignore this
+                    break
+                else:
+                    raise KeyError("Config file must define %s" % entry)
 
     if "ci_scripts" in config:
         for ci_config in config["ci_scripts"]:
             validate_ci_config(ci_config)
 
+    if "license_rst" in config:
+        check_list(config["license_rst"], "append")
```

Comment:
Is this key actually valid? I can't see it referenced in LICENSE.rst.template, did it get replaced by extending the template?

Reply:
Ah yeah that's been removed, forgot to take that part out.

Reply:
(sidenote, there are a bunch of other keys that we could be applying this check to, but I haven't gone through and done that systematically)